.github/workflows/alas-issues.yaml

```diff
@@ -0,0 +1,26 @@
+---
+name: "[ALAS] Open issues for new bulletins"
+on:
+  workflow_dispatch:
+    inputs:
+      window:
+        description: "Only consider bulletins published within this relative time window (golang Duration)"
+        default: "24h"
+        required: true  
+  schedule:
+    # once an hour, at the top of hour
+    - cron: "0 0 * * *"
```

Comment:
Do we have confidence this won't miss anything? Every 24 look at that past 24.

Reply:
Every *hour* look at the past 24 hours. So even if GitHub Actions has a lengthy outage that upends all executions, we're very unlikely to miss bulletins (and can always run it manually with a different value to backfill -- that's what the `workflow_dispatch` enables).